examples/mixture/plot_bayesian_gaussian_mixture.py

```diff
@@ -0,0 +1,115 @@
+"""
+==============================================
+Bayesian Gaussian Mixture Alpha Prior Analysis
+==============================================
+
+Plot the resulting ellipsoids of a mixture of three Gaussians with EM and
+variational Bayesian Gaussian Mixture for three different values of the  prior
+of alpha.
+
+For all models, the Variationnal Bayesian Gaussian Mixture adapts it number of
+mixture automatically. The parameter alpha prior has a direct link with the
+resulting number of components. Specifying a high value of alpha leads more
+often to uniformly-sized mixture components, while specifying small (under 0.1)
```

Comment:
`alpha prior has a direct link with the resulting number of components'.
I think you should mention which case (small/large alpha) tends to produce larger/smaller number of components.
alpha prior -> alpha_prior